vocabularies/geological-sites.ttl

```diff
@@ -138,32 +143,69 @@ gsqst:colluvial a skos:Concept ;
 
 gsqst:outcrop a skos:Concept ;
     skos:broader gsqst:field-site ;
-    dcterms:source "Merriam-Webster Dictionary" ;
+    dct:source "Merriam-Webster Dictionary" ;
     skos:definition "The part of a rock formation that appears at the surface of the ground."@en ;
```

Comment:
formation has stratigraphic meaning, and the def implies that the outcrop is only where the ground surface meets a rock exposure, and that rocks above the surface of the ground are not part of the outcrop.

suggested def. A field site where bedrock is exposed at the surface of the earth, and not concealed by regolith or water. There is a high degree of confidence that samples and observations from an outcrop site relate to the bedrock at that location.

Reply:
done